### PR TITLE
Updated the color of chatbot icon

### PR DESCRIPTION
**Pull request by BAHLEMAJOLA**
Updated styling on the chatbot interface for a more polished and consistent look.
Adjusted color scheme to better match the overall app theme.
Tweaked font sizes and spacing for improved readability
Updated the color of chatbot icon to match download CV button

# Before
before

# After
after

### CODE REVIEW

**Comment by zolwethu** [comment]
Looks cleaner with the overall visuals of the page

File: css/bot.css [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+    z-index: 100;
+    bottom: 20px;
+    right: 20px;
+    background-color: #0073e6; /* hot pink */
+    padding: 15px;
+    border-radius: 50%;
+    cursor: pointer;
+    color: white;
+    box-shadow: 0 4px 10px rgba(0, 0, 0, 0.1);
+}
```
Comment (TshepoMagagula) [inline css/bot.css:15]: Maybe you can add a "click to chat" message when a user hovers over the chatbot Icon

File: css/bot.css [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+.chat-box {
+    padding: 10px;
+    height: 270px;
+    overflow-y: auto;
+    background-color: #fff;
+    border-radius: 0 0 10px 10px;
+}
+
+/* Bot Message */
+.bot-message {
```
Comment (TshepoMagagula) [inline css/bot.css:69]: The chatbot works well and it looks good

**Comment by NuffSaid-Bore** [comment]
Screenshot 2024-10-31 191816
Try to reduce font side of Bahle's Assistant so that the bot profile shows up as well as to make the close button visible. It's right on top of the bot's face.

**Comment by NuffSaid-Bore** [comment]
Try to use the background color on the contact me button as the bot's background color just to harmonize colors, especially if on the landing page.

File: css/bot.css (hunk 1 above)
Comment (zolwethu) [reply css/bot.css:15]: Arigato, user expirience is more imersive not

Comment (zolwethu) [reply css/bot.css:15]: now**

**Comment by zolwethu** [comment]
oh n bahle try to chence the scroll button from black to the same color as the chat background

**Comment by Sinawo** [comment]
> Screenshot 2024-10-31 191816 Try to reduce font side of Bahle's Assistant so that the bot profile shows up as well as to make the close button visible. It's right on top of the bot's face.

I did reduce it to 14px.

**Pull request merged**